### DISCUSSION TITLE
使用atlas精简系统后，启用rime输入法后，点击任务栏三角折叠图标后，右键输入法图标后资源管理器闪退重启

### DISCUSSION THREAD

**Discussion by Readfere** [Atlas-OS/Atlas#1183] [Help & Support]
使用atlas精简系统后，启用rime输入法后，点击任务栏三角折叠图标后，右键输入法图标后资源管理器闪退重启
系统版本：<img width="348" alt="iux4rzmb y24" src="https://github.com/user-attachments/assets/e71627ba-6edf-474f-a025-d36a1f17fa1e">

错误应用程序名称: Explorer.EXE，版本: 10.0.22621.3880，时间戳: 0x0a9e5890错误模块名称: InputHost.dll，版本: 10.0.22621.3810，时间戳: 0x3da8a9e3异常代码: 0xc0000409错误偏移量: 0x000000000005d296错误进程 ID: 0x0x4B48错误应用程序启动时间: 0x0x1DAEC9C5F5F511E错误应用程序路径: C:\WINDOWS\Explorer.EXE错误模块路径: C:\Windows\System32\InputHost.dll报告 ID: 82a7b659-f192-4c6c-8c5a-653386343d47错误程序包全名: 错误程序包相对应用程序 ID:

**Comment by pencilnav** [comment]
@Readfere 
Hi there, If possible, please provide more details about your problem. You can use software like OBS to record your screen activities or provide detailed step-by-step actions with screenshots included. This way, we can better identify the issue. Thanks.

你好，如果可以的话，麻烦提供多一点信息供我们更好的排查你的问题。你可以使用录屏软件（例如OBS）录制你的操作，或提供附带截图的详细操作步骤。谢谢。
